bin/lensinit.js

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env node
+
+/**
+ * bin/lens-init.js
```

Comment:
camelCase the file name, and the name here in the comment should match the file name